### ISSUE TITLE
crash while launch

### ISSUE THREAD

**Issue by superiums** [phase1geo/Minder#328]
crash while launch,  happens afer upgrade in theses two moths. (reinstall not work)

version: 1.12.0
os: arch Linux

it reports:
``` bash
I/O warning : failed to load external entity "/home/user/.local/share/minder/favorites.xml"
I/O warning : failed to load external entity "/home/user/.local/share/minder/exports.xml"
free(): invalid pointer
fish: Job 1, 'com.github.phase1geo.minder' terminated by signal SIGABRT (Abort)
```

**Comment by phase1geo** [comment]
This is a duplicate issue of #318

**Comment by phase1geo** [comment]
BTW, I have just submitted a fix to the master branch for that issue.  If you would like to build Minder from the GitHub master branch to verify the issue is fixed, I will close this issue.

**Comment by phase1geo** [comment]
I have released 1.12.1 that should fix this issue. Closing bug.
